Assets/MRTK/SDK/Editor/Inspectors/UX/ScrollingObjectCollection/ScrollingObjectCollectionInspector.cs

```diff
@@ -276,6 +276,7 @@ private void DrawDebugSection()
                         if (check.changed)
```

Comment:
Currently the target is not being marked as dirty by the editor even using PropertyField.